pom.xml

```diff
@@ -230,15 +220,6 @@
 				</plugins>
 			</build>
 		</profile>
-		<profile>
-			<id>doclint-disable</id>
```

Comment:
no longer any need to handle this as a profile since we're no longer juggling java 8 in the mix.